flink-python/src/main/java/org/apache/flink/client/python/PythonDriverEnvUtils.java

```diff
@@ -256,4 +328,17 @@ public static Process startPythonProcess(PythonEnvironment pythonEnv, List<Strin
 
 		return process;
 	}
+
+	public static Process launchPy4jPythonClient(
```

Comment:
The name is confusing as it actually starts up a Python process and the main entrypoint of the Python process is one user's Python program instead of some script provided by Py4j. So what about rename it to prepareAndStartPythonProcess or revert this abstraction if we cannot come up with a meaningful name? Besides, what about moving this method to PythonDriver?